### ISSUE TITLE
Visual Problems with Unverified Email Banner and Verified Email Page

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/Pilcrow#1971]
**Describe the bug**
The unverified email banner uses white text on the warning brand color resulting in a failing 1.69 contrast ratio. 

The landing page for verifying an email address has a strange layout. The email icon of the landing page does not align with the adjacent descriptive text. The 'Dashboard' link is shoved into this section further contributing to the alignment issues of this landing page.

**To Reproduce**
1. Register a new user on Pilcrow.
2. Notice that the unverified email banner has an insufficient color contrast.
2. Go to MailHog (or whatever email inbox) and click the 'Verify Email Address' button within the Verify Email Address email.
3. On the resulting landing page, notice that the misalignment and lack of visual balance of elements on this landing page.

**Expected behavior**
The banner should have a minimum WCAG AA passing color contrast. 

The elements on the email verification landing page should have a strong and balanced alignment. 

**Screenshots**
[Screenshot of the email verification landing page showing that the unverified email banner with insufficient color contrast and the verified email landing page with a visually unbalanced and misaligned layout](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbVhsIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--d80130e5d2927bed22570175d45f9ec570bdaa0a/pilcrow.lndo.site_verify-email_1695867577_3bd25987acbd62826d88ab8d96e7859d727bffbb08bdb6c06a30138c35096a94%28iPad%20Air%29.png)
